### ISSUE TITLE
Unicode alias from → to -> in FSM

### ISSUE THREAD

**Issue by ktoso** [akka/akka#20233]
As noticed by @bwmcadams in https://github.com/akka/akka/pull/19746

**Comment by ktoso** [comment]
Need to fix it to unblock scalariform move.

**Comment by ktoso** [comment]
MiMa mislead me...

**Comment by patriknw** [comment]
Duplicate of https://github.com/akka/akka/issues/19772
